.github/workflows/testarch_device.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+export INTERCHANGE_SCHEMA_PATH="$GITHUB_WORKSPACE/env/fpga-interchange-schema/interchange"
+
+sudo apt-get install capnproto -y
```

Comment:
Probably this install step is not required as capnp is manually built and installed from the tarball.

Reply:
You are right, I've forgotten about it